src/ngraph/op/fused/unsqueeze.cpp

```diff
@@ -34,12 +34,27 @@ op::Unsqueeze::Unsqueeze(const Output<Node>& data, const Output<Node>& axes)
 
 void op::Unsqueeze::pre_validate_and_infer_types()
 {
+    auto data = input_value(0);
+    auto data_rank = data.get_partial_shape().rank();
     auto axes_node = input_value(1).get_node_shared_ptr();
 
-    // Currently only support Constant node for axes.
+    // Get value of axes from Constant
+    auto axes_constant = as_type_ptr<op::Constant>(axes_node);
+    auto axes = axes_constant->cast_vector<size_t>();
```

Comment:
This line is going to cause a segfault if `axes_node` is not a `Constant`. IMO the `NODE_VALIDATION_CHECK` that was here before should have stayed.